WinPort/src/Backend/TTY/TTYInputSequenceParserExts.cpp

```diff
@@ -491,7 +497,7 @@ size_t TTYInputSequenceParser::TryUnwrappWinMouseEscapeSequence(const char *s, s
 
 	if(args[2] > 0 && args[3] == 1){ // only KeyDown and valid char should pass
 		//fprintf(stderr, "Parsed: ==%c==\n", (unsigned char)(args[2]));
-		_win_mouse_buffer.push_back((unsigned char)args[2]);
+		_win_double_buffer.push_back((unsigned char)args[2]);
```

Comment:
Seems like unicode lost in here, when we cast to unsigned char. I designed it only for parse mouse sequence but now unicode points may occur here. 
maybe WCHAR cast needed here?

Reply:
Yes, fixed.